x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/bulk_snooze_modal.tsx

```diff
@@ -43,13 +47,29 @@ const failureMessage = i18n.translate(
   }
 );
 
+const deleteConfirmPlural = (total: number) =>
+  i18n.translate('xpack.triggersActionsUI.sections.rulesList.bulkUnsnoozeConfirmationPlural', {
+    defaultMessage: 'Unsnooze all {total, plural, one {# rule} other {# rules}}? ',
```

Comment:
IMO "all" isn't necessary here:
```suggestion
    defaultMessage: 'Unsnooze {total, plural, one {# rule} other {# rules}}? ',
```